examples/official-site/sqlpage/migrations/01_documentation.sql

```diff
@@ -764,6 +764,7 @@ INSERT INTO parameter(component, name, description, type, top_level, optional) S
     ('javascript', 'The URL of a Javascript file to load and execute on the page.', 'URL', TRUE, TRUE),
     ('rss', 'The URL of an RSS feed to display in the top navigation bar. You can use the rss component to generate the field.', 'URL', TRUE, TRUE),
     ('image', 'The URL of an image to display next to the page title.', 'URL', TRUE, TRUE),
+    ('image_og', 'The URL of an image to display in the Open Graph metadata.', 'URL', TRUE, TRUE),
```

Comment:
Less technical users will probably not know what "Open Graph metadata" means. We could explain that it's the preview image that will appear if the page is shared on social media.

Reply:
We should still mention open graph, but add the explanation